client/state/notices/actions.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * Internal dependencies
+ */
+import {
+	NEW_NOTICE,
+	REMOVE_NOTICE
+} from 'state/action-types';
+
+import { uniqueId } from 'lodash';
```

Comment:
We should not be importing from Lodash this way, as it increases each bundle size by about ~500kb. See #735 for a detailed explanation.